HacktoberfestProject.Web/Views/Shared/_Login.cshtml

```diff
@@ -1,10 +1,18 @@
-﻿<ul class="navbar-nav">
-	@if (!User.Identity.IsAuthenticated)
-	{
-		<a asp-action="Login" asp-controller="Auth" class="btn btn-outline-primary">Log In with GitHub</a>
-	} else
-	{
-		<a asp-action="Logout" asp-controller="Auth" class="btn btn-outline-primary">Logout</a>
-	}
+﻿@if (!User.Identity.IsAuthenticated)
```

Comment:
If the only difference for this if-statement is the text and location of the link, we could optimize and not have the svg in there twice. Downside: we have (something that looks like) an if-statement twice.

Pseudo-code:

```
<a ... asp-action="@(User.IsdAuthenticated ? "Logout" : "Login"") ... />
```

and

```
@(User.IsAuthenticated ? "Logout" : "Log in with GitHub")
```